festim/hydrogen_transport_problem.py

```diff
@@ -164,6 +167,14 @@ def temperature_time_dependent(self):
     def multispecies(self):
         return len(self.species) > 1
 
+    @property
+    def volume_subdomains(self):
+        return [s for s in self.subdomains if isinstance(s, F.VolumeSubdomain)]
+
+    @property
+    def surface_subdomains(self):
+        return [s for s in self.subdomains if isinstance(s, F.SurfaceSubdomain)]
+
```

Comment:
Is this even used anywhere now?

Reply:
The volume subdomains is still used in a few places, surface_subdomains no, could remove that one